app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,18 @@
+class LikesController < ApplicationController
+  def create
+    user = User.find(params[:user_id])
+    post = Post.find(params[:post_id])
+    like = Like.new(user:, post:)
+    respond_to do |format|
+      format.html do
+        if like.save
+          like.counter_updater
+          flash[:success] = 'Liked'
+        else
+          flash.now[:error] = 'Error: Like could not be saved'
+        end
+        redirect_to user_post_likes_path
+      end
+    end
+  end
+end
```

Comment:
- [ ] [Optional] Same here, Please use strong parameters in this [lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.) and that's to prevent any unwanted parameters added when creating a new like.

Reply:
Adding strong parameters is not working for me. I will try to add it in the subsequent PRs. Thank you